spec/awesome_spawn_spec.rb

```diff
@@ -7,10 +7,10 @@
   shared_examples_for "run" do
     context "options" do
       it ":params won't be modified" do
-        params      = {:user => "bob"}
+        params      = {:params => {:user => "bob"}}
         orig_params = params.dup
-        allow(subject).to receive(:launch).and_return(["", "", 0])
-        subject.send(run_method, "true", :params => params)
+        allow(subject).to receive(:launch).with("true --user bob", {}).and_return(["", "", 0])
+        subject.send(run_method, "true", params)
```

Comment:
This actually changes the original spec.  Before it would test the inner hash wasn't modified, now it no longer tests that.